### PR TITLE
Add a header to prevent a fatal error

### PR DESCRIPTION
**Pull request by kienstra**
Fixes an issue a user raised to support: that this sometimes causes a fatal error on installing via `wp-admin/plugin-install.php`

### How to test

1. Go to `/wp-admin/plugin-install.php`
2. Search for Genesis Simple Hooks
Screenshot 2023-09-29 at 12 14 40 PM

3. Install and activate it
4. Expected: The page loads
5. Actual: There's sometimes an error, though you might not see it:

Screenshot 2023-09-29 at 12 13 31 PM

That's because the header [was missing](https://github.com/studiopress/genesis-simple-hooks/pull/37#discussion_r1341831830).

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -2,13 +2,42 @@ version: 2.1
...
   php: circleci/php@1.0
-  wp-svn: studiopress/wp-svn@0.1
+  wp-svn: studiopress/wp-svn@0.2
 
+references:
+  PLUGIN_SLUG: &PLUGIN_SLUG
+    genesis-simple-hooks
+
+jobs:
+  zip:
```
Comment (kienstra) [inline .circleci/config.yml:12]: This `zip` job lets us test the wp.org deployment without doing the deployment.

We'll get the `.zip` in the job's Artifacts tab.

It uses the same `.distignore`, so we can see if that allows the files it should (like `.php` files), and ignores the files it should (like `composer.lock` and `package.json`).

File: .distignore [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+.distignore
+.git
+.gitattributes
+.gitignore
+composer.json
+composer.lock
+Gruntfile.js
+package.json
+package-lock.json
+node_modules
```
Comment (kienstra) [inline .distignore:10]: Changed from `.svnignore` so we can also use this in the `zip` job

File: genesis-simple-hooks.php [hunk 3]
```diff
@@ -1,6 +1,13 @@
...
 /**
- * This file handles the creation of the Simple Hooks admin menu.
+ * Plugin Name: Genesis Simple Hooks
+ * Plugin URI: https://studiopress.com
+ * Description: Creates a new Genesis settings page that allows you to insert code (HTML, Shortcodes, and PHP), and attach it to any of the 50+ action hooks throughout the Genesis Theme Framework, from StudioPress.
+ * Author: StudioPress
+ * Author URI: https://www.studiopress.com/
+ * Version: 2.3.1
+ * License: GPL2+
+ * License URI: http://www.gnu.org/licenses/gpl-2.0.txt
```
Comment (kienstra) [inline genesis-simple-hooks.php:10]: This header was missing, and caused an error on installing from wp.org:

Screenshot 2023-09-29 at 12 13 31 PM

File: package.json [hunk 4]
```diff
@@ -15,7 +15,7 @@
     "description": "Genesis Simple Hooks allows you easy access to the 50+ Action Hooks in the Genesis Theme.",
     "author": "StudioPress",
     "authoruri": "http://www.studiopress.com/",
-    "version": "2.3.0",
+    "version": "2.3.1",
```
Comment (kienstra) [inline package.json:18]: If it's alright, I'm bumping the version here, so we don't need a separate PR for it.

**Comment by kienstra** [comment]
Related to https://wordpress.org/support/topic/bad-header-it-wont-work/

**Review by mike-day — CHANGES_REQUESTED** [review]
Thanks for taking on this ticket, @kienstra!

I noticed that there is a file called `plugin.php` that also has a header. That file also includes a `require_once` for `genesis-simple-hooks.php`, but that shouldn't be needed.

Maybe that was needed for a very old version of WP, but it is definitely not needed anymore!

---

The presence of both headers causes the plugin to show up twice for me locally:

Screenshot 2023-10-02 at 4 50 59 PM

[This PR](https://github.com/studiopress/genesis-simple-hooks/pull/38) deletes `plugin.php` and adds a few header rows from that file to `genesis-simple-hooks.php`.

File: package.json (hunk 4 above)
Comment (mike-day) [reply package.json:18]: Nice! Great idea to prevent a separate, tiny PR 😎

**Review by mike-day — APPROVED** [review]

**Comment by dreamwhisper** [comment]
@mike-day @kienstra Took me a bit to find why we kept the plugin.php files, but I found Nathan's comment [here](https://github.com/studiopress/genesis-simple-sidebars/pull/36#issuecomment-486264270) about breaking on update.

**Comment by kienstra** [comment]
Thanks, @dreamwhisper!

I'm not going to merge this.

Corinne didn't see an improvement from this PR.

**Pull request closed without merging**